compiler/parser/chapel.ypp

```diff
@@ -1866,8 +1870,14 @@ parenthesized_expr:
 | TLP tuple_expr_ls TCOMMA TRP      { $$ = buildTuple($2); }
 ;
 
+bool_literal:
+  TFALSE { $$ = new SymExpr(gFalse); }
+| TTRUE  { $$ = new SymExpr(gTrue); }
+;
+
 literal_expr:
-  INTLITERAL            { $$ = buildIntLiteral($1, yyfilename, chplLineno);    }
+  bool_literal
```

Comment:
I never remember what Bison's default action is so would tend to put `{ $$ = $1; }` here but it doesn't bother me if you can remember and rely on the default.